content/doc/book/installing/_docker.adoc

```diff
@@ -322,7 +322,7 @@ As mentioned <<accessing-the-jenkins-console-log-through-docker-logs,above>>,
 your `<docker-container-name>` can be obtained using the
 link:https://docs.docker.com/engine/reference/commandline/container_ls/[`docker container ls`]
 command. If you specified the +
```

Comment:
I think that the root of the problem is the trailing space on the end of the preceding line.  A hard line break is inserted for the example line that is not inserted in the previous use of the same technique.

```suggestion
command. If you specified the
```

Reply:
I would remove the hard line break since the example code is not continue. It's broken in two pieces separated by `option in the`.